sdk/servicebus/service-bus/src/util/runtimeInfo.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
`core-http` has similar logic already.  You can use `getDefaultUserAgentValue()` from `core-http`, then just need to add the prefix for service bus.

Reply:
I did this for storage https://github.com/Azure/azure-sdk-for-js/pull/9519/files

Reply:
Interesting, I just copied this from event-hubs and was wondering if I should refactor it into a common package like core-amqp.

event-hubs doesn't depend on core-http, at least I'll use `getDefaultUserAgentValue` for service-bus.

Reply:
Lets log an issue and put it in the backlog to refactor both Event Hubs and Service Bus.
For the scope of this PR, let's keep these changes

Reply:
On second thoughts, event hubs does not have a dependency on core-http, so perhaps we can move the common code to core-amqp as part of the new issue

Reply:
`getDefaultUserAgentValue` adds `core-http` in the user agent string, which we wouldn't want for AMQP operations.

`getPlatformSpecificData` in `core-http` is the equivalent of the `getRuntimeInfo` in this PR, but `getPlatformSpecificData` is not exported from core-http.

So, I'll do
- use the `getDefaultUserAgentValue` for the ATOM management client
- retain the `getRuntimeInfo` for the AMQP operations
- in a future PR, move `getRuntimeInfo` from service-bus and event-hubs to core-amqp

Reply:
#10140 logged to move the common code to core-amqp